please.sh

```diff
@@ -4775,20 +4775,24 @@ find_mspdb_dll () { #
 	return 1
 }
 
-build_mingw_w64_git () { # [--only-32-bit] [--only-64-bit] [--skip-test-artifacts] [--skip-doc-man] [--skip-doc-html] [--force] [<revision>]
+build_mingw_w64_git () { # [--only-i686] [--only-x86_64] [--only-aarch64] [--skip-test-artifacts] [--skip-doc-man] [--skip-doc-html] [--force] [<revision>]
 	output_path=
 	sed_makepkg_e=
 	force=
 	src_pkg=
 	while case "$1" in
-	--only-32-bit)
+	--only-i686|--only-32-bit)
 		MINGW_ARCH=mingw32
 		export MINGW_ARCH
 		;;
-	--only-64-bit)
+	--only-x86_64|--only-64-bit)
```

Comment:
I left `--only-64-bit` in place to not break existing CI pipelines and scripts. Ideally these should move to `--only-x86_64` too.